catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageWebConverter.java

```diff
@@ -0,0 +1,1760 @@
+            InternationalStringType istToPopulate =
+                    RegistryPackageUtils.getInternationalStringTypeFromString(valueToPopulate);
+            if (istToPopulate != null) {
+                registryObject.setDescription(istToPopulate);
+                populated = true;
+            }
+        }
+
+        if (map.containsKey(EXTERNAL_IDENTIFIER_KEY)) {
+            populateExternalIdentifierList((List<Map<String, Object>>) map.get(
+                    EXTERNAL_IDENTIFIER_KEY), registryObject.getExternalIdentifier());
+            if (!registryObject.getExternalIdentifier()
+                    .isEmpty()) {
+                populated = true;
+            }
+        }
+
+        valueToPopulate = getStringFromMap(HOME_KEY, map);
+        if (StringUtils.isNotBlank(valueToPopulate)) {
+            registryObject.setHome(valueToPopulate);
+            populated = true;
+        }
+
+        valueToPopulate = getStringFromMap(ID_KEY, map);
+        if (StringUtils.isNotBlank(valueToPopulate)) {
+            registryObject.setId(valueToPopulate);
+            populated = true;
+        }
+
+        valueToPopulate = getStringFromMap(LID_KEY, map);
+        if (StringUtils.isNotBlank(valueToPopulate)) {
+            registryObject.setLid(valueToPopulate);
+            populated = true;
+        }
+
+        valueToPopulate = getStringFromMap(NAME_KEY, map);
+        if (StringUtils.isNotBlank(valueToPopulate)) {
+            InternationalStringType istToPopulate =
+                    RegistryPackageUtils.getInternationalStringTypeFromString(valueToPopulate);
+            if (istToPopulate != null) {
+                registryObject.setName(istToPopulate);
+                populated = true;
+            }
+        }
+
+        valueToPopulate = getStringFromMap(OBJECT_TYPE_KEY, map);
+        if (StringUtils.isNotBlank(valueToPopulate)) {
+            registryObject.setObjectType(valueToPopulate);
+            populated = true;
+        }
+
+        if (map.containsKey(SLOT)) {
+            populateSlotList((List<Map<String, Object>>) map.get(SLOT), registryObject.getSlot());
+            if (!registryObject.getSlot()
+                    .isEmpty()) {
+                populated = true;
+            }
+        }
+
+        valueToPopulate = getStringFromMap(STATUS_KEY, map);
+        if (StringUtils.isNotBlank(valueToPopulate)) {
+            registryObject.setStatus(valueToPopulate);
+            populated = true;
+        }
+
+        valueToPopulate = getStringFromMap(VERSION_INFO_KEY, map);
+        if (StringUtils.isNotBlank(valueToPopulate)) {
+            VersionInfoType versionInfo = RIM_FACTORY.createVersionInfoType();
+            versionInfo.setVersionName(valueToPopulate);
+
+            registryObject.setVersionInfo(versionInfo);
+            populated = true;
+        }
+
+        return populated;
+    }
+
+    private static String getStringFromMap(String key, Map<String, Object> map) {
+        String value = null;
```

Comment:
`map.get()` already returns `null` if the key doesn't exist. Should this method do this instead?
```
String value = map.get();

return (value instanceof String) ? value : null;
```
Or maybe we need to throw an exception and log something if the value isn't a String?